pandas/tests/arrays/boolean/test_function.py

```diff
@@ -54,7 +54,8 @@ def test_ufuncs_binary(ufunc):
     tm.assert_extension_array_equal(result, expected)
 
     # not handled types
-    with pytest.raises(TypeError):
+    msg = r"operand type\(s\) all returned NotImplemented from \_\_array\_ufunc\_\_"
```

Comment:
why are the underscores escaped?

Reply:
@simonjayhawkins 
Oh, it's my fault. I fix it.